easy-ui-tokens/src/font/base.json

```diff
@@ -16,6 +16,7 @@
       "50": { "value": "12px", "attributes": { "unit": "rem" } },
       "75": { "value": "13px", "attributes": { "unit": "rem" } },
       "100": { "value": "15px", "attributes": { "unit": "rem" } },
+      "125": { "value": "16px", "attributes": { "unit": "rem" } },
```

Comment:
i feel a little uneasy about the button text being 1px bigger than the body text 🤔 did we confirm that that's indeed what design wants?